src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -5,40 +5,47 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Arrays;
+import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class StreamStudy {
 
-    public static long countWords() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
-    }
-
-    public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        // TODO 이 부분에 구현한다.
-    }
-
-    public static List<Integer> doubleNumbers(List<Integer> numbers) {
-        return numbers.stream().map(x -> 2 * x).collect(Collectors.toList());
-    }
-
-    public static long sumAll(List<Integer> numbers) {
-        return numbers.stream().reduce(0, (x, y) -> x + y);
-    }
-
-    public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
-    }
+	public static long countWords() throws IOException {
+		String contents = new String(Files.readAllBytes(Paths
+			.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+		return Arrays.stream(contents.split("[\\P{L}]+"))
+			.filter(word -> word.length() > 12)
+			.count();
+	}
+
+	public static List<String> printLongestWordTop100() throws IOException {
+		String contents = new String(Files.readAllBytes(Paths
+			.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+
+		// TODO 이 부분에 구현한다.
+		return words.stream()
+			.filter(word -> word.length() > 12)
+			.sorted(Comparator.comparingInt(String::length))
```

Comment:
요구사항이 단어의 길이가 긴 순서대로 출력하는 것 이기에 정렬 비교시 다음과 같이 추가해주시면 좋을 것 같아요 :)
```suggestion
			.sorted(Comparator.comparingInt(String::length).reversed())
```

Reply:
제가 제대로 확인을 못했네요.. 챙겨주셔서 감사합니다 ㅎ